lib/Sema/CSSimplify.cpp

```diff
@@ -2040,6 +2040,19 @@ bool ConstraintSystem::repairFailures(
     return false;
   };
 
+  auto repairByAddingConformance = [&](Type lhs, Type rhs) -> bool {
+    if (lhs->isTypeVariableOrMember())
+      return false;
+
+    if (rhs->isAny() ||
+        !(rhs->is<ProtocolType>() || rhs->is<ProtocolCompositionType>()))
```

Comment:
!rhs->isExistentialType()

Reply:
Yeah, I think it makes sense to test that too.

Reply:
Sorry, I mean literally isExistentialType() is equivalent to checking for ProtocolType and ProtocolCompositionType :)

Reply:
Ah, thanks! I am going to change that in CSDiag as well then!